Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "UICommon/CommandLineParse.h"
+#include "Common/Common.h"
```

Comment:
I don't think anything from this file needs `Common.h`. `Common/Version.h` is likely better.